MAUI/Toolbar/toolbar-items.md

```diff
@@ -0,0 +1,1541 @@
+
+The spacing between the toolbar items is equalized when the `ItemSpacing` property is set to -1.
+ 
+## Item Alignment
+
+The Toolbar control provides support for customizing the alignment of the toolbar items using the `ItemAlignment` property. This property allows you to set the alignment of the toolbar items to `Start`, `Center`, or `End`.
+
+The following code sample demonstrates how to Create a Toolbar control with Item Alignment.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:local="clr-namespace:ToolbarSample"
+             xmlns:toolbar="clr-namespace:Syncfusion.Maui.Toolbar;assembly=Syncfusion.Maui.Toolbar"
+             x:Class="ToolbarSample.MainPage">
+        <ContentPage.BindingContext>
+            <local:ToolbarViewModel/>
+        </ContentPage.BindingContext>
+         <toolbar:SfToolbar x:Name="Toolbar"
+                        HeightRequest="56"
+                        Items="{Binding ToolbarItems}">
+         </toolbar:SfToolbar>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+using Syncfusion.Maui.Toolbar;
+
+namespace ToolbarSample
+{
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+        }
+    }
+
+    public class ToolbarViewModel
+    {
+        public ObservableCollection<BaseToolbarItem> ToolbarItems { get; set; }
+        
+        public ToolbarViewModel()
+        {
+            ToolbarItems = new ObservableCollection<BaseToolbarItem>()
+            {
+                new SfToolbarItem() { Alignment = ToolbarItemAlignment.Start, Icon = new FontImageSource() { Glyph = "\uE713", FontFamily = "MaterialAssets" }, Text = "Start Item" },
+                new SeparatorToolbarItem() { Alignment = ToolbarItemAlignment.Start },
+                new SfToolbarItem() { Alignment = ToolbarItemAlignment.Center, Icon = new FontImageSource() { Glyph = "\uE714", FontFamily = "MaterialAssets" }, Text = "Center Item" },
+                new SeparatorToolbarItem() { Alignment = ToolbarItemAlignment.Center },
+                new SfToolbarItem() { Alignment = ToolbarItemAlignment.End, Icon = new FontImageSource() { Glyph = "\uE715", FontFamily = "MaterialAssets" }, Text = "End Item" },
+                new SeparatorToolbarItem() { Alignment = ToolbarItemAlignment.End },
+            };
+        }
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+ 
+## Leading and Trailing Item
+
+The Toolbar control provides support for setting leading and trailing items using the `ItemAlignment` property. The items set with `ItemAlignment` as `Start` will be displayed at the leading position, and the items set with `ItemAlignment` as `End` will be displayed at the trailing position. By default, the items will be displayed in the center position.
+
```

Comment:
add the note

N> The leading and trailing is applicable when the toolbar is not scrollable mode.